main.js

```diff
@@ -97,7 +101,7 @@ const showcaseCheckout = array => {
   })
   const itemTotalCount = createElement('div', {class: 'item-total ion-pound'}, ["Total Items: " + array.length])
   const itemTotalPrice = createElement('div', {class: 'item-total ion-social-bitcoin-outline'}, ["Total: " + getItemTotal(cartArray)])
-  const submitCheckout = createElement('button', { class: 'submit-checkout'}, ["Checkout!"])
+  const submitCheckout = createElement('button', { onclick: "clicked()", type: "submit", class: 'submit-checkout', href: "#"}, ["Checkout!"])
```

Comment:
We don't add inline click handlers in vanilla JavaScript. You should be adding a `submit` event listener to your checkout view container (just like our other event listeners).